devtools/__main__.py

```diff
@@ -24,7 +26,7 @@ def print_code() -> int:
 def install() -> int:
     print('[WARNING: this command is experimental, report issues at github.com/samuelcolvin/python-devtools]\n')
 
-    if 'debug' in __builtins__.__dict__:
+    if 'debug' in builtins.dir():
```

Comment:
```suggestion
    if hasattr(builtins, 'debug'):
```
is faster man arguable more canonical.